apis/slo/v1alpha1/nodeslo_types.go

```diff
@@ -328,6 +328,10 @@ type ResourceThresholdStrategy struct {
 	// +kubebuilder:validation:Maximum=100
 	// +kubebuilder:validation:Minimum=0
 	CPUSuppressThresholdPercent *int64 `json:"cpuSuppressThresholdPercent,omitempty" validate:"omitempty,min=0,max=100"`
+	// cpu suppress min percentage (0,100)
+	// +kubebuilder:validation:Maximum=100
+	// +kubebuilder:validation:Minimum=0
+	CPUSuppressMinPercent *int64 `json:"cpuSuppressMinPercent,omitempty" validate:"omitempty,min=0,max=100"`
```

Comment:
define default value in DefaultResourceThresholdStrategy

Reply:
https://github.com/koordinator-sh/koordinator/issues/2371